catalog/app/utils/Store.js

```diff
@@ -25,21 +24,18 @@ export const Provider = function StoreProvider({ initialState = {}, history, chi
       process.env.NODE_ENV === 'development' &&
       process.env.LOGGER_REDUX === 'enabled'
     ) {
-      const stateTransformer = (state) =>
-        // pure JS is easier to read than Immutable objects
-        Iterable.isIterable(state) ? state.toJS() : state
       // eslint-disable-next-line global-require
       const { createLogger } = require('redux-logger')
-      middlewares.push(createLogger({ stateTransformer, collapsed: true }))
+      middlewares.push(createLogger({ stateTransformer: R.identity, collapsed: true }))
```

Comment:
do we still need `stateTransformer` parameter here?

Reply:
I don't know. And I don't refactor code in this PR, only remove Immutable.js

Reply:
well, you can remove it to avoid future confusion as to why was it kept

Reply:
I shouldn't but I can remove it if you would be more polite